src/publisher/tests/test_utils.py

```diff
@@ -21,7 +21,7 @@ def test_state_error(self):
             # bit naff
             self.assertTrue(err.trace.strip().startswith('File "'))
             self.assertTrue(
-                err.trace.strip().endswith("raise utils.StateError(1, \"msg\")")
+                err.trace.strip().endswith("""raise utils.StateError(1, \"msg\")""")
```

Comment:
this is due to an interaction with the linter Black and the needs to test for an escaped message